### ISSUE TITLE
[BUG] Missing aria-labels on back button in page template and close button in modal/drawer

### ISSUE THREAD

**Issue by MortenGregersen** [kirbydesign/designsystem#905]
**Describe the bug**
The back button on the page template has no text and therefore needs to have an `aria-label`. The same is the case for the close button on the drawer and the modal.

**Kirby version**
- 0.8.87

**Expected behavior**
The buttons should have a `aria-label` set to be accessible.

**Additional context**
I don't know how this should be done. The texts needs to be localised, and I don't know if Kirby should be localised to handle this, or the texts can be supplied in any way.

**Comment by RasmusKjeldgaard** [comment]
This should be covered in https://github.com/kirbydesign/designsystem/issues/942.
